eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalSplicesStateSpec.scala

```diff
@@ -1725,13 +1777,15 @@ class NormalSplicesStateSpec extends TestKitBaseClass with FixtureAnyFunSuiteLik
     awaitCond(alice.stateData.asInstanceOf[DATA_NORMAL].commitments.active.size == 1)
     awaitCond(bob.stateData.asInstanceOf[DATA_NORMAL].commitments.active.size == 1)
 
-    resolveHtlcs(f, htlcs, spliceOutFee = 0.sat)
+    resolveHtlcs(f, htlcs, 0 sat)
```

Comment:
nit: `resolveHtlcs` should consistently show the param name `spliceOutFee`, or never show it, right now it's inconsistently shown.